src/multipart.lua

```diff
@@ -214,7 +224,13 @@ function MultipartData.new(data, content_type)
   local instance = setmetatable({}, MultipartData)
 
   if content_type then
-    instance._boundary = match(content_type, ";%s*boundary=(%S+)")
+    local boundary = match(content_type, ";%s*boundary=(%S+)")
```

Comment:
I don't know how big of a deal this is, but if someone sends a malformed header which does not contain "boundary" (`multipart/form-data; limit=AaB03x`) this match will return nil and the following `if` will throw an error. You might want to consider detecting the case and returning an error here.

Reply:
yes, let's do that! definitely!